### ISSUE TITLE
Case-insensitive find icon is confusing

### ISSUE THREAD

**Issue by mcclurgm** [elementary/code#934]
## What Happened

This is basically just opinion based, but I find the icon we use for case insensitive find confusing. 
I suppose the capital letter implies some sort of case sensitivity, but that's not very clear to me. To me, the arrows on either side of the letter look like it should represent whole word search, not case sensitive (I was about to submit a bug report about weird behavior in whole word search until I saw the tooltip 🙃️). Since whole word is a common feature in other text editors, I think it's fair to say this is confusing and something we should design around.

## Expected Behavior

The icon should more clearly imply case sensitivity, and shouldn't be as easily confused with whole word.

Since this isn't really a bug, none of the rest of the fields really apply…

**Comment by cassidyjames** [comment]
Yeah we should design and gresource a more semantic icon, or add it to the system icons if it's something we want to be standard.

**Comment by danirabbit** [comment]
Fixed in #1276
